### PR TITLE
Prepare 1.3 release

### PR DESCRIPTION
**Pull request by DimCitus**
Checklist: https://github.com/citusdata/pg_auto_failover/issues/252

### CODE REVIEW

**Comment by JelteF** [comment]
I think this should be mentioned:
https://github.com/citusdata/pg_auto_failover/pull/212

**Comment by DimCitus** [comment]
> I think this should be mentioned:
> #212

done now, thanks!

File: src/bin/lib/libs/ini.h [hunk 1]
```diff
@@ -486,7 +486,6 @@ ini_t* ini_create( void* memctx )
     ini->sections[ 0 ].name[ 0 ] = '\0'; 
     ini->sections[ 0 ].name_large = 0;
     ini->properties = (struct ini_internal_property_t*) INI_MALLOC( ini->memctx, INITIAL_CAPACITY * sizeof( ini->properties[ 0 ] ) );
-	memset(ini->properties, 0, INITIAL_CAPACITY * sizeof(ini->properties[0]));
```
Comment (JelteF) [inline src/bin/lib/libs/ini.h:489]: This seems like a change that we introduced. I guess with a reason.

Comment (DimCitus) [reply src/bin/lib/libs/ini.h:489]: isn't that https://github.com/mattiasgustavsson/libs/commit/5d00587c897f7992c8bd948ca1f44938325bade4#diff-3d315c8a064a9c199059040b5edc4dc7 ?

Comment (DimCitus) [reply src/bin/lib/libs/ini.h:489]: mmm that said I'm not sure after all, did our change come from the security work stream and review?

Comment (DimCitus) [reply src/bin/lib/libs/ini.h]: thanks for the link to https://github.com/citusdata/citus-ha/commit/d3ccd755dc4c534e7356f2508f42a1597e7be934 ; cancelled my change.

File: src/bin/pg_autoctl/pgsql.c [hunk 2]
```diff
@@ -860,7 +860,18 @@ pgsql_replication_slot_drop_removed(PGSQL *pgsql, NodeAddressArray *nodeArray)
 		" SELECT slot_name, pg_drop_replication_slot(slot_name) "
 		"   FROM pg_replication_slots pgrs LEFT JOIN nodes USING(slot_name) "
 		"  WHERE nodes.slot_name IS NULL "
+		"    AND (   slot_name ~ '" REPLICATION_SLOT_NAME_PATTERN "' "
+		"         OR slot_name ~ '" REPLICATION_SLOT_NAME_DEFAULT "' )"
```
[the remark this answers is not in the document]
Comment (DimCitus) [reply src/bin/pg_autoctl/pgsql.c:843]: Turns out there's no call sites of this function's only call site. Removed from the code.

**Pull request merged**